content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -149,6 +149,10 @@ scroll to the :guilabel:`Colombian Electronic Invoicing` section. Then, follow t
 #. Configure the |DIAN| environment; the |DIAN| electronic invoicing module offers three different
    |DIAN| environments to connect with:
 
+   - **DIAN Demo Mode**: This environment allows to test electronic invoicing workflows using
+     a demo digital certificate. Files are created and accepted automatically and are not sent 
+     to any DIAN environment. Every internal validation can be tested.
+
```

Comment:
Slight RST syntax adjustments and adding a note just in case users have the test env activated
```suggestion
   - **DIAN demo mode**: This environment allows to test electronic invoicing workflows using
     a demo digital certificate. Files are created and accepted automatically and are not sent
     to any |DIAN| environment. Every internal validation can be tested. To activate it, tick the
     :guilabel:`DIAN Demo Mode` checkbox.

     .. note::
        The :guilabel:`DIAN Demo Mode` checkbox is hidden from view when the :guilabel:`Test
        environment` checkbox is enabled.
```